murfi-rt-PyProject/scripts/feedback.sh

```diff
@@ -58,43 +67,34 @@ then
 
     # CCCB version (direct flirt from subject functional to MNI structural: step 1)
     # because the images that we get from Prisma through Vsend are in LPS orientation we need to change both our MNI mean image and our mni masks accordingly: 
-   fslswapdim MNI152_T1_2mm.nii.gz x -y z MNI152_T1_2mm_LPS.nii.gz
-   fslorient -forceneurological MNI152_T1_2mm_LPS.nii.gz
-#   once the images are in the same orientation we can do registration
+    fslswapdim MNI152_T1_2mm.nii.gz x -y z MNI152_T1_2mm_LPS.nii.gz
+    fslorient -forceneurological MNI152_T1_2mm_LPS.nii.gz
+    # once the images are in the same orientation we can do registration
     rm -r $subj_dir/xfm/epi2reg
     mkdir $subj_dir/xfm/epi2reg
     mkdir $subj_dir/mask/lps
 
-#for mni_mask in {dmn,cen,smc}; #include this for DMN feedback
-    for mni_mask in {dmn,cen,smc,stg};do 
+    # For each MNI mask in the participant's MNI mask directory
```

Comment:
@pab2163 yes, this will reorient and transform into the current subject space the mni templates, it iterates over the specified masks. we could keep smc in there for our future sham r33 part. we need to double check how the images are acquired to see if we need the LPS step or not. at northeastern we needed this.